### ISSUE TITLE
13 pin DIN connector

### ISSUE THREAD

**Issue by RFmanNJ** [zarpli/PS-WTX500#1]
You did is an awesome work!
On the DIN connector shown below:
![](http://i.imgur.com/miE0v.png)

How and what did you connect your the two RCA connectors?
Did you connect SW and FL speakers only from the DIN connection to RCA connectors? Or did you connect to 13P.ST ACON1 connector in your subwoofer?

I am about to do this to my subwoofer as well.

Thank you,

**Comment by zarpli** [comment]
hi,
you can send the analog audio signals through the DIN connector, I do not have the DIN connector cable, that is why I connected the audio signals directly to the PCB.

Please share photos or video of your hack :)

**Comment by RFmanNJ** [comment]
Definitely, you did all the work. I am just following your foot step. I appreciate your response. I will take a lot of pictures. I didn't know how to contact you. So, I posted on the issue.

You are the man!

**Comment by RFmanNJ** [comment]
Zarpli,
I didn't record video. But I was successfully implement the code and modified the subwoofer. I only tried the subwoofer but it's working like a charm! Great work.

**Comment by zarpli** [comment]
I'm glad you did the hack, I hope later you can share a photo, best regards
